plugin/src/main/kotlin/net/octyl/ts2kt/gradle/repository/npm/NpmClientRepository.kt

```diff
@@ -42,11 +43,12 @@ import net.octyl.ts2kt.gradle.repository.dependency.ClientDependency
 import net.octyl.ts2kt.gradle.repository.dependency.ExternalClientDependency
 import net.octyl.ts2kt.gradle.util.PartialPackageInfo
 import org.gradle.api.Project
+import org.gradle.internal.os.OperatingSystem
```

Comment:
Can this be done without touching `internal` API? I prefer to stay away from it if possible. It looks like `org.apache.tools.ant.taskdefs.condition.Os` may be a good replacement.

Reply:
I don't remember finding a way either, this package seemed to be the common practice. If you figure out how, by all means.

Reply:
org.apache.tools.ant.taskdefs.condition.Os requires Ant api, which would be worse than an 'internal' API